core/src/main/scala/cats/Monad.scala

```diff
@@ -162,6 +164,18 @@ trait Monad[F[_]] extends FlatMap[F] with Applicative[F] {
     tailRecM(branches.toList)(step)
   }
 
+  /**
+   * If the `F[Boolean]` is `true` then return `ifTrue` otherwise return `Monoid[A].empty`
+   */
+  def ifTrueM[B: Monoid](fa: F[Boolean])(ifTrue: => F[B]): F[B] =
+    ifM(fa)(ifTrue, pure(Monoid[B].empty))
+
+  /**
+   * If the `F[Boolean]` is `false` then return `ifFalse` otherwise return `Monoid[A].empty`
+   */
+  def ifFalseM[B: Monoid](fa: F[Boolean])(ifFalse: => F[B]): F[B] =
+    ifM(fa)(pure(Monoid[B].empty), ifFalse)
```

Comment:
I'd suggest adding these methods to the `F[Boolean]` syntax in Mouse https://github.com/typelevel/mouse/blob/main/shared/src/main/scala/mouse/fboolean.scala